ipatests/test_integration/test_installation.py

```diff
@@ -392,6 +393,20 @@ def get_pki_tomcatd_pid(host):
     return(pid)
 
 
+def get_ipa_services_pids(host):
+    ipa_services_name = [
+        "krb5kdc", "kadmin", "named", "httpd", "ipa-custodia",
+        "pki_tomcatd", "ipa-dnskeysyncd"
+    ]
+    pids = []
```

Comment:
Instead of building a list, I would use a dict with key=service name, value=pid.